example/src/main/java/io/netty/example/http2/helloworld/client/Http2Client.java

```diff
@@ -106,16 +108,17 @@ public static void main(String[] args) throws Exception {
 
             HttpResponseHandler responseHandler = initializer.responseHandler();
             int streamId = 3;
-            URI hostName = URI.create((SSL ? "https" : "http") + "://" + HOST + ':' + PORT);
+            HttpScheme scheme = SSL ? HttpScheme.HTTPS : HttpScheme.HTTP;
```

Comment:
No. They are used to directly set the ":scheme" pseudo header. I don't think this should https://tools.ietf.org/html/rfc7540#section-8.1.2.3 have anything appended?

The hostname was incorrect before and also should not have the scheme etc...